app/src/main/java/com/babylon/wallet/android/domain/NetworkContent.kt

```diff
@@ -0,0 +1,11 @@
+package com.babylon.wallet.android.domain
+
+sealed class NetworkContent<out T> {
```

Comment:
that's actually a good idea!  💡 
my only objection is that it's not a domain model, but a presentation one... thus only in viewmodels is used, and it would fit under the `presentation/common/` package